sk0-back/src/utils/mailer.service.ts

```diff
@@ -0,0 +1,29 @@
+import * as nodemailer from "nodemailer";
```

Comment:
Not sure if we wanted to avoid using the while whole module import or not.

Reply:
Okay 👍